units/XNL0111/XNL0111_Script.lua

```diff
@@ -23,15 +23,19 @@ XNL0111 = Class(NAmphibiousUnit, SlowHover) {
         },
     },
 
+    ---@param self XNL0111
     OnCreate = function(self)
         NAmphibiousUnit.OnCreate(self)
         --save the modifier for max radius so we dont have to go into the blueprint every time.
         local wep = self:GetWeaponByLabel('MainGun')
-        local bp = wep:GetBlueprint()
+        local bp = wep.Blueprint
```

Comment:
`bp` usually refers to the bp of `self` in FAF code.
```suggestion
        local weaponBp = wep.Blueprint
```